code-challenges.test.js

```diff
@@ -13,15 +13,36 @@
 // --------------------1) Create a function that takes in a string and returns a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
 
 // a) Create a test with expect statements using the variables provided.
-
+describe("mixedCode", () => {
+    it("takes in letters and returns numbers instead", () => {
 const secretCodeWord1 = "Lackadaisical"
+expect()
 // Expected output: "L4ck4d41s1c4l"
 const secretCodeWord2 = "Gobbledygook"
 // Expected output: "G0bbl3dyg00k"
 const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
+
 // b) Create the function that makes the test pass.
+// if it senses an a whether its A or a it will instead send 4
+// it will do similar process for the other letters
+const mixedCode = codeArr.map((value) => {
+   if
+(value === "A" || value === "a") {
+    return "4";
+  } else if (value === "E" || value === "e") {
+    return "3";
+  } else if (value === "I" || value === "i") {
+    return "1";
+  } else if (value === "O" || value === "o") {
+    return "0";
+  } else {
+    return value;
+  }
+    })
+})
```

Comment:
I like your logic.  However, your code is set to map an array, but the input is a string.  Also, your test is incomplete.  Be sure to run your code to see if it's working.  If not tell us what you tried.